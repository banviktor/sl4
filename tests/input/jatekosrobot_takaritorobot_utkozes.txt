randomization off
questions off
start_game 2
5 5
5 1
toggle_glue
jump
inputvector 0 5
jump
I
6.1 5
0 -5
jump
inputvector 0 5
jump
N
list_cleaner_robots
list_smudges
exit
