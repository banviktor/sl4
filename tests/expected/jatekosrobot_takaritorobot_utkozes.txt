Véletlenszerű események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Ragacs be.
Olajfolt ki.
Ragacsfolt lehelyezve.
Új pozíció: 5.0, 5.0
Ez a(z) 1. kör.
A Sárga Robot következik.
Inputvector: 0.0, 1.0
Új pozíció: 5.0, 2.0
Takarítórobot indul útnak innen: 6.1, 5.0
Ez a(z) 2. kör.
A Piros Robot következik.
Új pozíció: 5.0, 5.0
Ez a(z) 2. kör.
A Sárga Robot következik.
Inputvector: 0.0, 1.0
Új pozíció: 5.0, 4.0
Ez a robot a következőkkel ütközött:
	Takarítórobot
Ez a(z) 3. kör.
A Piros Robot következik.
Takarítórobotok listája:
Foltok listája:
	Ragacs: 5.0, 5.0
	Olaj: 5.1, 4.0
